src/commands/install.ts

```diff
@@ -335,7 +349,12 @@ export async function install(
       }),
       rollupPluginCss(),
       rollupPluginCommonjs({
-        extensions: ['.js', '.cjs'], // Default: [ '.js' ]
+        extensions: ['.js', '.cjs'],
+        // Workaround: CJS -> ESM isn't supported yet by the plugin, so we needed
+        // to add our own custom workaround here. Requires a fork of
+        // rollupPluginCommonjs that supports the "externalEsm" option.
+        // @ts-ignore
```

Comment:
I understand why this is here, but is there a way instead to do `{ … } as RollupCommonJSOptions`? Sometimes `@ts-ignore` can hide other problems unintentionally